narwhal/types/src/primary.rs

```diff
@@ -315,6 +337,24 @@ impl From<HeaderDigest> for Digest<{ crypto::DIGEST_LENGTH }> {
     }
 }
 
+impl From<HeaderDigest> for Digest<{ crypto::INTENT_MESSAGE_LENGTH }> {
```

Comment:
this is used for signature service to accept an intent message (a digest of intent || the hash)